packages/esm-patient-medications-app/src/medications-summary/medications-summary.component.tsx

```diff
@@ -21,7 +21,7 @@ export default function MedicationsSummary({ patient }: MedicationsSummaryProps)
     error: error,
     isLoading: isLoading,
     isValidating: isValidating,
-  } = usePatientOrders(patient?.id, 'any');
+  } = usePatientOrders(patient?.id);
 
   const [pastOrders, activeOrders] = useMemo(() => {
```

Comment:
Use the past orders and active orders from `useActivePatientOrders` and `usePastPatientOrders`. You don't need `usePatientOrders` at all in this file.